ngraph/frontend/ir/src/model.cpp

```diff
@@ -651,8 +651,11 @@ std::shared_ptr<ngraph::Function> XmlDeserializer::parse_function(const pugi::xm
     //  Following topological order create nGraph operations
     for (auto& layer_id : order) {
         auto& p = params[layer_id];
-        ngraph::OutputVector inputs(edges[layer_id].size());
-        for (auto& e : edges[layer_id]) {
+        auto& edge = edges.find(layer_id);
```

Comment:
Shouldn't it be handled by ngraph team?

Reply:
The task is to handle all KW hits in related components